app/models/user.rb

```diff
@@ -23,9 +23,18 @@ def to_param
   end
 
   def open_items
-    self.items.where(status: "open")
+    self.items.open.distinct
   end
 
+  def won_items
```

Comment:
Is there another way to retrieve unique items here? A user has many items through bids